cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/ExemploVmAllocationPolicyMigrationWorstFitStaticThreshold.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.cloudsimplus.migration;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Set;
+import org.cloudbus.cloudsim.allocationpolicies.power.PowerVmAllocationPolicyMigrationStaticThreshold;
+import org.cloudbus.cloudsim.hosts.Host;
+import org.cloudbus.cloudsim.hosts.power.PowerHost;
+import org.cloudbus.cloudsim.hosts.power.PowerHostSimple;
+import org.cloudbus.cloudsim.selectionpolicies.power.PowerVmSelectionPolicy;
+import org.cloudbus.cloudsim.vms.Vm;
+import org.cloudsimplus.sla.readJsonFile.SlaMetric;
+import org.cloudsimplus.sla.readJsonFile.SlaReader;
+
+/**
+ *
+ * @author raysaoliveira
+ */
+public class ExemploVmAllocationPolicyMigrationWorstFitStaticThreshold extends PowerVmAllocationPolicyMigrationStaticThreshold {
```

Comment:
The class was duplicated from a core class and must be deleted.
The method VerifyCpuMetricMinThreshold is already duplicated in a previous class, as commented before.